src/admin/shared/components/ContentPicker/ContentPicker.helpers.ts

```diff
@@ -1,19 +1,22 @@
 import { get } from 'lodash';
 import { ValueType } from 'react-select/src/types';
 
-import { ContentPickerType, PickerItem, PickerSelectItem, PickerTypeOption } from '../../types';
+import { ContentPickerType } from '@viaa/avo2-components';
 
+import { PickerItem, PickerSelectItem, PickerTypeOption } from '../../types';
+
+// TODO: add PROFILE to ContentPickerType
```

Comment:
Is er hiervoor een PR in de typings? Dan moet je hier typings en de typings release vermelden die dit zal fixen